API/functions/index.js

```diff
@@ -59,13 +59,19 @@ messaging_api.get('/test', async (request, response) => {
   response.json({"test2":"message"});
 });
 
+//check for new messages of uid
+messaging_api.get('/getMessages', async (request, response) => {
+  const req_key = request.get('auth');
```

Comment:
You should probably get the uid from the query params and pass it to getMessages()